src/Illuminate/Cache/ArrayStore.php

```diff
@@ -173,7 +173,7 @@ public function getPrefix()
      * Get the expiration time of the key.
      *
      * @param  int  $seconds
-     * @return int
+     * @return float
```

Comment:
This is a breaking change

Reply:
Yes and no. The methods are protected, and the expiresAt value is not accessible outside. If someone has already redefined methods, we won't break anything. If people haven't redefined the methods, then we won't break anything either.

These are already implementation details, but it seems to me that there are enough changes every time. I don’t need these changes, I just updated the PR from @timacdonald

Reply:
Changing the return type of a protected method is still a breaking change.

Reply:
Well then there are two options:
- close PR;
- transfer changes to laravel 11;

As I already wrote, I don’t need these changes. I brought the changes so that if you come back to make changes to ArrayCache, I don’t have to deal with debugging and come to you with an issue #48569 )

Reply:
I had a brilliant idea. And let me just bring a unit test for ArrayCache and add it. This will allow you to quickly find an error from the issue, and we will close this PR

Reply:
#48591